src/IconMgr.cpp

```diff
@@ -13,7 +13,7 @@ void WiconMgr::grabaIcono(char* fileName){
 
 Os2IconMgr12* WiconMgr::codecOs2Icon12(){
    Os2IconMgr12 *iconoWarp = new Os2IconMgr12();
-   //Se parte de la imagen mas peque�a para que el WPS asuma la imagen mas grande al desplegarla
+   //Se parte de la imagen mas peque¤a para que el WPS asuma la imagen mas grande al desplegarla
```

Comment:
What's the difference?

Reply:
GitHub changed the encoding to UTF. No optical difference for me.

Reply:
Then, remove these differences, please...

Reply:
GitHub changed the encoding. Nothing I can do about it...

Feel free to close this PR. Maybe you can correct the single typo in line 28.

Reply:
I suspect CRLF vs LF problem. Anyway I'll  fix a typo and close this PR. 

Thanks!